[2024-06-25T16:38:51.390+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-25T16:38:51.410+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: nasa_neo_data_to_s3.transform_nasa_data scheduled__2024-06-23T00:00:00+00:00 [queued]>
[2024-06-25T16:38:51.416+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: nasa_neo_data_to_s3.transform_nasa_data scheduled__2024-06-23T00:00:00+00:00 [queued]>
[2024-06-25T16:38:51.416+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 4
[2024-06-25T16:38:51.427+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): transform_nasa_data> on 2024-06-23 00:00:00+00:00
[2024-06-25T16:38:51.433+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=74) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-25T16:38:51.434+0000] {standard_task_runner.py:63} INFO - Started process 78 to run task
[2024-06-25T16:38:51.433+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'nasa_neo_data_to_s3', 'transform_nasa_data', 'scheduled__2024-06-23T00:00:00+00:00', '--job-id', '76', '--raw', '--subdir', 'DAGS_FOLDER/etl_neo.py', '--cfg-path', '/tmp/tmp_sbxb2xq']
[2024-06-25T16:38:51.437+0000] {standard_task_runner.py:91} INFO - Job 76: Subtask transform_nasa_data
[2024-06-25T16:38:51.475+0000] {task_command.py:426} INFO - Running <TaskInstance: nasa_neo_data_to_s3.transform_nasa_data scheduled__2024-06-23T00:00:00+00:00 [running]> on host 95b3b965c5b1
[2024-06-25T16:38:51.537+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='nasa_neo_data_to_s3' AIRFLOW_CTX_TASK_ID='transform_nasa_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-23T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-23T00:00:00+00:00'
[2024-06-25T16:38:51.538+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-25T16:38:51.581+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-25T16:38:51.582+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-25T16:38:51.589+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=nasa_neo_data_to_s3, task_id=transform_nasa_data, run_id=scheduled__2024-06-23T00:00:00+00:00, execution_date=20240623T000000, start_date=20240625T163851, end_date=20240625T163851
[2024-06-25T16:38:51.608+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-25T16:38:51.626+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-25T16:38:51.628+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
